New package for the CIM v16 and Dynamics

fran-jo opened the pull request:
Created a package Dynamics. In this first step, we only created classes for SynchronousMachineDynamics.
TODO: The rest of the packages have to be moved to a parent package Base
Script for reading CIM models have been modified to read the new namespace / version of CIM. Right now it reads one type of SynchronousMachineDynamics

fran-jo committed:
New package for the CIM v16 and Dynamics

rwl commented:
Python 2.7 support must be maintained. Please check RDFXMLReader.py encoding issue:

https://travis-ci.org/rwl/PyCIM/jobs/261895577

fran-jo committed:
Setup modified to support new package CIM16

Rearrenging of packages for new organization within CIM16
Rearrenging of packages to test new namespaces within CIM14

fran-jo committed:
Change name to test file
